### DISCUSSION TITLE
What should we call the database?

### DISCUSSION THREAD

**Discussion by SamKimball-BrownU** [arch-kiosk/arch-kiosk-office#1221] [General]
Kiosk is the name of the server and synchronization management system. What should we call the actual database? We have been calling it FileMaker, but that is actually the piece of software that we use on the back end to manage it. So, there is a bit of a mix up of terms.

So, what should we call the piece of software that users interact with most in the field? Taking any and all ideas.

**Comment by myjaylee1123** [comment]
Thought: NCARS - New and Cool Archaeological Recording System

**Comment by urapadmin** [comment]
this is more complicated than it seems, There are reasons to call it simply something like filemaker field recording or so. Here are some background thoughts:
- kiosk is designed to handle different recording systems. I could write one one in Microsoft Access or even in Microsoft Excel or what have you. If we get the funding, the next step would be to create a browser based cross-platform recording that is really universal and much more integrated with kiosk. But it would still be handled and synchronized by kiosk.
- I would not want to make it a product because there can even be other filemaker recording systems managed by kiosk. Our current filemaker recording tries to be as universal as possible but if I cannot for my life make a project fit into it there is always the option to "just" develop a simpler, bespoke recording in filemaker and kiosk would manage it. So what we see as the filemaker recording system is sort of just one instance of a range of possibilites (a rather elaborate one, for sure). But I still don't want to make it more than it is by giving it a name. What I definitely have in mind is to build a smaller filemaker recording only for pedestrian survey that can run on phones. It would coexist with our current recording system. Another thought is that we might split the current filemaker database into separate parts. That would make the parts smaller. There is no reason for pedestrian survey to be in the same database with excavation. They could be different databases and kiosk would control by means of the workstation or some such which filemaker database gets prepared. We could separate things like the inventory, too. And if we do that there is no single piece of software anymore. 
- "recording" is essential to make clear that this is the thing for recording and not for other things. A bit like a field calculator which has a very limited use and is not the computer running GIS. In fact our current filemaker recording goes farther than it should (and one day we will cut it down, indeed).
- this is not the recording system I would want to develop if somebody asked me to develop a recording system. Since the beginning it has always been a bit like a very laborious attempt to find out what a good recording system would be. And this isn't it :o) I always perceive it as temporary and provisional and even though I know that provisional solutions can last very long. they rarely deserve a too individual name. That is why I would go with something rather descriptive.

**Comment by lbestock** [comment]
Despite the clear and well-reasoned argument for a descriptive name, I find that tonight I want to name the whole recording system, and all possible iterations in different platforms that might be managed by kiosk: Tractatus. But that in the background, rarely used but known by the initiates. And then pieces of it one gets from Kiosk I want to name after candy bars. Our current recording system could be Twix. We get a Twix (which is a branch of Tractatus) from the Kiosk. As one does. Someone wants just survey? Snickers. Someone brings their own bespoke excel sheet with no philosophy behind it? Goobers.

**Comment by urapadmin** [comment]
Nobody ever gave me a goobers in the states. Is that edible?

**Comment by myjaylee1123** [comment]
Oh, I like the thought of naming it after candy - it's a lot like the way Android names its software.

**Comment by lbestock** [comment]
Theoretically Goobers should be edible - chocolate covered peanuts. But it sounds so much like boogers (they MUST have known that, right? marketing target 7 year old boys?) that I've never even tried them.

**Comment by cmrice-arch** [comment]
Also never had Goobers because yeah, boogers... nothing about the name, shape, packaging seems appealing. Who does eat these??

**Comment by luizaogs** [comment]
I thought Laurel had made them up? America is a strange place.

But then I looked them up because I was appalled and curious in equal measure and apparently "Goober" derives from "guber," which is the Gullah word for "peanut." Should have picked a different name, clearly.

**Comment by SamKimball-BrownU** [comment]
Sauce

Because candy is out, may I propose sauce as the name for the database? Would be better if we could come up with some contrived acronym... Simple Archaeological... User... C???? E?????

We could then name other spinoff programs after various other types of sauce. Béchamel as a survey tool?
